src/video/v9990/V9990.cc

```diff
@@ -569,6 +581,7 @@ void V9990::writeRegister(byte reg, byte val, EmuTime::param time)
 	assert(reg < 64);
 	if (!(regAccess[reg] & ALLOW_WRITE)) {
 		// register not writable
+		invalidRegisterWriteCallback.execute(reg, val);
```

Comment:
Ah, the write callback takes two parameters. Good idea.
But then the documentation above should reflect that (currently it only talks about the register parameter).

BTW now that I see the patch, I agree it's simpler in c++ than via a Tcl script.